### PR TITLE
Cannot get available browser processes on Windows (#22)

### LINKED ISSUE #22: Cannot get available browser processes on Windows
It seems there is a bug at line 11 of [windows.js](https://github.com/sindresorhus/kill-tabs/blob/main/windows.js#L11) script. 

The call from `promisify(childProcess.exec)` at line 9 will return an object with the following properties: `stderr`, `stdout`, `error`.

To be able succesffully call `execall` we need to access the `stdout` value.  

I'm pushing a fix in a few minutes. I will also be a good idea to check for `stderr` and report accordingly.

### PR DESCRIPTION
**Pull request by GeorgeGkas**
Fixes #22

### CODE REVIEW

**Comment by sindresorhus** [comment]
Linting is failing. Make sure you always run `npm test` locally.

**Comment by GeorgeGkas** [comment]
updated

File: windows.js [hunk 1]
```diff
@@ -6,9 +6,14 @@ const TEN_MEBIBYTE = 1024 * 1024 * 10;
 
 export default async function killTabs() {
 	const command = 'wmic process where Caption=\'chrome.exe\' get CommandLine,ProcessId /format:list';
-	const stdout = await promisify(childProcess.exec)(command, {maxBuffer: TEN_MEBIBYTE});
+	const response = await promisify(childProcess.exec)(command, {maxBuffer: TEN_MEBIBYTE});
 
-	return execall(/CommandLine=(.+)\s+ProcessId=(\d+)/g, stdout).map(element => ({
+	if (response.stderr) {
+		console.error(response.stderr);
+		throw new Error('Failed to kill tabs.');
```
Comment (sindresorhus) [inline windows.js]: Can you pass `stderr` (wrapped in an error) as the `Error#cause`?

https://2ality.com/2021/06/error-cause.html

Comment (GeorgeGkas) [reply windows.js]: you mean like

```js
if (response.stderr) {
    throw new Error(response.stderr);
}
```

???

Comment (sindresorhus) [reply windows.js]: No, please re-read my comment and check out the link.

Comment (GeorgeGkas) [reply windows.js]: From my understanding we wrap the existing error message by `stderr` into another error. The new error define as  `#cause`  the message from `stderr`. So something like the following if I got it correctly??

```js
throw new Error(`Failed to kill tabs.`, {cause: new Error(stderr)});
```

Comment (sindresorhus) [reply windows.js]: Correct

**Pull request merged**